tomviz/OperatorPropertiesPanel.cxx

```diff
@@ -145,4 +158,38 @@ void OperatorPropertiesPanel::apply()
     }
   }
 }
+
+void OperatorPropertiesPanel::viewCodePressed()
+{
+  if (!m_activeOperator) {
+    return;
+  }
+
+  if (m_activeOperator->hasCustomUI()) {
+    // See if we already have a dialog open for this operator
+    auto dialog =
+      ActiveObjects::instance().getDialogForOperator(m_activeOperator);
```

Comment:
I would much rather call m_activeOperator->customDialog() or similar here, rather than use active objects as the operator should maintain that mapping internally (unless I am missing something).